src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -10889,10 +10898,14 @@ void ObjectMgr::LoadCreatureStaticFlagsOverride()
             continue;
         }
 
-        if (std::find(creatureData->spawnDifficulties.begin(), creatureData->spawnDifficulties.end(), difficultyId) == creatureData->spawnDifficulties.end())
+        // DIFFICULTY_NONE is always a valid fallback
+        if (difficultyId != DIFFICULTY_NONE)
         {
-            TC_LOG_ERROR("sql.sql", "Table `creature_static_flags_override` has data for a creature that is not available for the specified DifficultyId (SpawnId: {}, DifficultyId: {}), skipped", spawnId, difficultyId);
-            continue;
+            if (std::find(creatureData->spawnDifficulties.begin(), creatureData->spawnDifficulties.end(), difficultyId) == creatureData->spawnDifficulties.end())
```

Comment:
You could use std::ranges instead

Reply:
That's from C++23 so we can't use it yet

Reply:
You're right, just set, map and other containers have .contains method.